src/labelle/lib/devices/device_config.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import annotations
+
+import logging
+from typing import Tuple
+
+LOG = logging.getLogger(__name__)
+
+
+class DeviceConfig:
+    """Configuration object for the capabilities of a label printer."""
+
+    name: str
+    """Name of this device"""
+
+    deviceIDs: list[int]
+    """List of USB Device ID's this device can identify as"""
+
+    printHeadSizePixels: int
+    """Size of the print head in pixels (use calibration routine to determine)"""
+
+    supportedTapeSizes: list[int]
+    """List of supported tape sizes in mm"""
+
+    marginsPerTape: dict[int, Tuple[int, int]]
+    """
+    Dictonary of print margins per tape size
+    Entry: TapeSizeMM : (topMarginInPx, bottomMarginInPx)
+    Use the calibration routine to determine these for each tape size
+    """
+
+    def __init__(
+        self,
+        name: str,
+        deviceIDs: list[int],
```

Comment:
Please avoid using camelCase, and instead use smake_case. For example `device_ids`, not `deviceIDs`.

https://stackoverflow.com/questions/159720/what-is-the-naming-convention-in-python-for-variables-and-functions

Reply:
Sorry. If you check the link you've sent you know what my default programming language is 😉 
I'll to a cleanup/refactor before it gets merged.

Reply:
(should be) fixed